backend/src/modules/language/model/language.output.ts

```diff
@@ -0,0 +1,18 @@
+import { ObjectType, Field, ID } from 'type-graphql';
+
+@ObjectType()
+export class LanguageOutput {
+  private _type = 'output';
+
+  @Field(type => ID)
+  id?: number;

```

Comment:
I wonder if we shouldn't use some conventional id (e.g. ISO 639-1) instead of a generated one.

Reply:
Nvm as commented for alpha3.